anagrafica/permessi/funzioni.py

```diff
@@ -309,6 +311,25 @@ def permessi_responsabile_autoparco(sede):
     ]
 
 
+def permessi_delegato_campagne(sede):
+    from donazioni.models import Campagna
+    filtro_sedi = Q(estensione__in=[NAZIONALE, TERRITORIALE])
```

Comment:
Grazie! Mi ero già documentato dato che me ne avevi già accennato. Ancora non ho avuto modo di duplicare quel codice, in effetti:) Comodo l'utilizzo di via() per usare la query con altri modelli.